config/database.yml

```diff
@@ -22,4 +22,5 @@ test:
 
 production:
   <<: *default
+  adapter: postgresql
   database: db/production.sqlite3
```

Comment:
Is this the right database?  Also do we need to support the environment variable being passed in by docker here?

Reply:
Env is already supported.